scripts/docker_setup.sh

```diff
@@ -98,23 +100,74 @@ function move_ifaces() {
 			sudo ip netns exec pause ethtool -N "${ifaces[$i]}" flow-type tcp4 action 0
 			sudo ip netns exec pause ethtool -u "${ifaces[$i]}"
 		fi
+		if [ "$mode" == 'cndp' ]; then
+			# num queues
+                        num_q=1
+                        # start queue index
+                        start_q_idx=22
+                        # RSS using TC filter
+			setup_tc "${ifaces[$i]}" $num_q $start_q_idx
+		fi
```

Comment:
Please properly format this file/changes

Reply:
Yes, I see this file has `tab` formatting, but FYI, lines 105-108 have spaces instead of `tab`. This comment also applies to `setup_tc()` and `add_tc_rules()` (they have spaces instead of `tab`)

Reply:
ok. Will update